### PR TITLE
[Hub Generated] Review request for Microsoft.ManagedServices to add version stable/2019-09-01

### PR DESCRIPTION
**Pull request by ajay-sainy**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/ajay-sainy/azure-rest-api-specs/dev-managedservices-Microsoft.ManagedServices-2019-09-01...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/managedservices/resource-manager/Microsoft.ManagedServices/stable/2019-09-01/managedservices.json [hunk 1]
```diff
@@ -0,0 +1,915 @@
...
+        },
+        "x-ms-examples": {
+          "Get Registration Assignments": {
+            "$ref": "./examples/GetRegistrationAssignments.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.ManagedServices/marketplaceRegistrationDefinitions/{marketplaceIdentifier}": {
+      "get": {
```
Comment (sanjaiganesh) [inline specification/managedservices/resource-manager/Microsoft.ManagedServices/stable/2019-09-01/managedservices.json]: ARM won't treat this as a collection call.. Taking offline.

Comment (ajay-sainy) [reply specification/managedservices/resource-manager/Microsoft.ManagedServices/stable/2019-09-01/managedservices.json]: Updated as per the discussions.

File: specification/managedservices/resource-manager/Microsoft.ManagedServices/stable/2019-09-01/managedservices.json [hunk 2]
```diff
@@ -812,6 +911,15 @@
       "type": "string",
       "description": "Scope of the resource."
     },
+    "MarketplaceIdentifier": {
+      "name": "marketplaceIdentifier",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method",
+      "x-ms-skip-url-encoding": true,
```
Comment (anthony-c-martin) [inline specification/managedservices/resource-manager/Microsoft.ManagedServices/stable/2019-09-01/managedservices.json]: Why skip URL encoding here?

Comment (ajay-sainy) [reply specification/managedservices/resource-manager/Microsoft.ManagedServices/stable/2019-09-01/managedservices.json]: That should not be there. Removed.

**Review by anthony-c-martin — APPROVED** [review]
LGTM

**Comment by ajay-sainy** [comment]
@zikalino Could you please review the PR?

**Review by zikalino — APPROVED** [review]

**Pull request merged**